plugins/modules/nodebalancer.py

```diff
@@ -523,6 +520,14 @@ def _update_nodebalancer(self) -> None:
         if "configs" in params.keys():
             params.pop("configs")
 
+        if "firewall_id" in params.keys():
+            firewall_id = params.pop("firewall_id")
+            fw = Firewall(self.client, firewall_id)
```

Comment:
I don't think this will make an API request immediately due to lazy-loading, so you might need to use `self.client.load(...)` instead